GET_STARTED.md

```diff
@@ -0,0 +1,442 @@
+
+To test a ccf application you need go through the following steps:
+
+- Start a CCF Network with at least one node
+- Initialize the CCF network with at least one (active member - user), this is done through [Network Governance Proposals](https://microsoft.github.io/CCF/main/governance/proposals.html).
+- Create an application [deployment proposal](https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html)
+- Submit the app deployment proposal to the network and all members accept it through voting, This is a part of [Network Governance](https://microsoft.github.io/CCF/main/governance/proposals.html).
+- Open the CCF network for users
+- Start to test your application endpoints
+
+### Build Application
+
+The application building prerequisites [[CCF](#ccf-install), NodeJS and NPM] must be installed,
+all will be preinstalled if you are using devcontainer environment,
+otherwise you need to manually install.
+
+In the checkout of this repository:
+
+```bash
+cd banking-app
+npm install
+# build and generate the application bundle and deployment proposal
+make build
+ls
+cd ..
+
+# a dist folder is created with app bundle.
+```
+
+---
+
+### Testing your Application
+
+There are servral ways and tools which helping to test your application
+
+- [Sandbox.sh](#testing-using-sandboxsh)
+
+  - Build an initialized CCF network and deploy your app on top of it
+  - Support both ccf network types [virtual - release (TEE hardware)]
+  - No governance steps required
+
+- [Docker container](#testing-using-docker-containers)
+
+  - Support both ccf network types [virtual - release (TEE hardware)]
+  - Governance steps required to deploy your app, initialize, and start the network
+
+- [Linux Machine](#testing-using-linux-machine)
+
+  - Support both ccf network types [virtual - release (TEE hardware)]
+  - Governance steps required to deploy your app, initialize, and start the network
+
+- [Azure Managed CCF Service](#testing-using-azure-managed-ccf-serivce-mccf)
+
+  - Using Azure MCCF service, will Create [a Managed CCF network](https://techcommunity.microsoft.com/t5/azure-confidential-computing/microsoft-introduces-preview-of-azure-managed-confidential/ba-p/3648986) which is (initialized, contains an active member, and In Open State)
+  - Support only a ccf network in release mode (TEE hardware)
+  - No governance steps required to start up your network, but you need to deploy your app proposal
+
+#### Testing: Using Sandbox.sh
+
+By running sandbox.sh script, it is automatically starts a CCF network and deploys your application on it.
+The app is up and ready to receive calls and all the governance work is done for you.
+
+Start in a CCF Network in Release mode
+
+```bash
+/opt/ccf/bin/sandbox.sh --js-app-bundle ./js/dist/  --enclave-type release -p /opt/ccf/lib/libjs.enclave.so.signed
```

Comment:
`./js/dist` - is this the right folder structure for the banking sample? It maybe correct for the ccf-app-template repo